src/objects/SkinnedMesh.js

```diff
@@ -134,7 +134,7 @@ THREE.SkinnedMesh.prototype.normalizeSkinWeights = function () {
 
 };
 
-THREE.SkinnedMesh.prototype.updateMatrixWorld = function( force ) {
+THREE.SkinnedMesh.prototype.updateMatrixWorld = function() {
 
 	THREE.Mesh.prototype.updateMatrixWorld.call( this, true );
```

Comment:
I expect it should be

    THREE.Mesh.prototype.updateMatrixWorld.call( this, force );

I wonder if this change will cause a problem?...